app/javascript/components/map/components/basemaps/basemaps.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import withTooltipEvt from 'components/ui/with-tooltip-evt';
+import { setMapSettings, setLandsatBasemap } from 'components/map/map-actions';
+import {
+  getBasemap,
+  getLabels,
+  getLayers,
```

Comment:
I think you renamed this.

Reply:
what do you mean?

Reply:
Didn't you rename this to getLayersDatasets?

Reply:
Ah I am wrong ignore